sdk/typescript/src/providers/provider.ts

```diff
@@ -20,18 +37,72 @@ export type TransactionResponse = string;
 // Exported Abstracts
 export abstract class Provider {
   // Objects
+  /**
+   * Get all objects owned by an address
+   */
   abstract getOwnedObjectRefs(address: string): Promise<ObjectRef[]>;
 
+  /**
+   * Get information about an object
+   */
+  abstract getObjectInfo(objectId: string): Promise<GetObjectInfoResponse>;
+
   // Transactions
+  /**
+   * Get transaction digests for a given range
+   *
+   * NOTE: this method may get deprecated after DevNet
```

Comment:
I agree that we will likely deprecate it: perhaps it would be better to move these into a different interface ?